cluster/saltbase/salt/kube-addons/kube-addon-manager.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   - name: kube-addon-manager
     # When updating version also bump it in:
     # - test/kubemark/resources/manifests/kube-addon-manager.yaml
-    image: gcr.io/google-containers/kube-addon-manager:v6.4-beta.2
+    image: gcr.io/danielmk-gke-dev/kube-addon-manager:v6.5
```

Comment:
We don't want to use private registries. This should come from google-containers.

Reply:
Sorry - leftover from testing. Changed to google-containers now. Thanks for catching this!